cjsmin/src/chess.ts

```diff
@@ -733,7 +766,10 @@ export class Chess {
             from: square,
             to: this._epSquare,
             piece: PAWN,
-            captured: PAWN,
+            capture: {
```

Comment:
and this? why capture vs captured

Reply:
originally I kept both capure & captured for backwards compatability. Didn't update the name again after I changed that